rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -1887,6 +1887,34 @@ static bool SetUnitResourceParam(CUnit* unit, const char* name, float value)
 }
 
 
+static bool SetUnitStorageParam(CUnit* unit, const char* name, float value)
+{
+	// [m|e]
+	//
+	//         metal | energy
+
+	// Hmmm wondering if there is a way to do it without copy.
```

Comment:
Copy is cheap, it's just two floats. Don't worry about it (even if the compiler doesn't optimize it out altogether)